enhancements/installer/vsphere-ipi-zonal.md

```diff
@@ -511,12 +734,107 @@ type VCenter struct {
 	// +kubebuilder:validation:MinItems=1
 	Datacenters []string `json:"datacenters"`
 }
+
+// Host defines host VMs to generate as part of the installation.
+type Host struct {
+	// FailureDomain refers to the name of a FailureDomain as described in https://github.com/openshift/enhancements/blob/master/enhancements/installer/vsphere-ipi-zonal.md
+	// +optional
+	FailureDomain string `json:"failureDomain"`
+	// NetworkDeviceSpec to be applied to the host
+	// +kubebuilder:validation:Required
+	NetworkDevice *NetworkDeviceSpec `json:"networkDevice"`
+	// Role defines the role of the node
```

Comment:
Please expand the godoc here, what do each of these values mean and what effect does it have when I set them?

Reply:
Same as above, this is what is currrent.